web/src/beta/features/Visualizer/Crust/Widgets/Widget/builtin/DataAttribution/UI/index.tsx

```
@@ -0,0 +1,126 @@
+                  >
+                    <Typography color="#000" size="body">
+                      {credit.description}
+                    </Typography>
+                  </StyledLink>
+                ) : (
+                  <Typography color="#000" size="body">
+                    {credit.description}
+                  </Typography>
+                )}
+              </ListItems>
+            ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)